backend/src/config/index.js

```diff
@@ -135,6 +135,8 @@ nconf.defaults({
       gradeCodeURL: process.env.INSTITUTE_API_URL + '/grade-codes',
       provinceCodesURL: process.env.INSTITUTE_API_URL + '/province-codes',
       countryCodesURL: process.env.INSTITUTE_API_URL + '/country-codes',
+      reportingRequirementCodesURL: process.env.INSTITUTE_API_ENDPOINT +
```

Comment:
use INSTITUTE_API_URL

Reply:
@SodhiA1 https://github.com/bcgov/EDUC-STUDENT-ADMIN/pull/1326/commits/5cb063f7a82f1661f1a2b891933ea65ae4c732a7 There it is! :)